tensorflow_datasets/core/dataset_info.py

```diff
@@ -42,12 +42,14 @@
 import tempfile
 
 from absl import logging
+import matplotlib.pyplot as plt
```

Comment:
Please use matplotlib as lazy deps

Reply:
Where to add matplotlib as lazy dep ?

Reply:
In: https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/core/lazy_imports.py

More info in: https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md#extra-dependencies

Reply:
Its already added - https://github.com/tensorflow/datasets/blob/2c43a797fec4fb10c246b229bc47c7b391346f8c/tensorflow_datasets/core/lazy_imports.py#L64

Reply:
But you still have to us it in your function, instead of global import.